api/src/routing.js

```diff
@@ -296,6 +296,7 @@ app.get('/extension-libs/:name', extensionLibs.get);
 app.get(routePrefix + 'login', login.get);
 app.get(routePrefix + 'login/identity', login.getIdentity);
 app.postJson(routePrefix + 'login', login.post);
+app.get(routePrefix + 'login/oidc', login.getSSOLogin);
```

Comment:
This causes a recursion because `login.getSSOLogin` should redirect to this endpoint.

Reply:
Remove this, it's added when addressing #9765

Reply:
PR updated @benkags